packages/cashier/src/stores/__tests__/payment-agent-store.spec.js

```diff
@@ -113,10 +111,7 @@ describe('PaymentAgentStore', () => {
     });
 
     it('should set active_tab_index then send verification email if it is equal to 1', () => {
-        payment_agent_store.verification.sendVerificationEmail = jest.fn();
-
         payment_agent_store.setActiveTab(1);
```

Comment:
Doesn't need to add any expect for this test?

Reply:
@hamid-deriv I'm not sure it's the old code, Should I add it? 🤔

Reply:
This test case is not testing anything right now after removing verification from the component. I think this test case can be removed.